ls_formatting_test.go

```diff
@@ -135,7 +135,7 @@ func runLsTestHelper(t *testing.T, result, expectedType, path string) {
 
 	// username / uid (len 8, number or string)
 	const (
-		name = "(?:[a-z_][a-z0-9_]*)"
+		name = "(?:[a-z_][-a-z0-9_]*)"
```

Comment:
If we’re going to be expanding this username regexp, then we should probably target adding all the allowed characters (that is, underscore and period) and upper-case letters.

Reply:
I updated the pull request to include those in the regexp too, thanks!